src/App.js

```diff
@@ -5,14 +5,12 @@ function App() {
   return (
     <div className="App">
       <header className="App-header">
-      Weidman Family Library <AddBook />
-        </header>
+        Weidman Family Library 
+        <AddBook />
+      </header>
```

Comment:
Happy to see some formatting! That was going to be one of the first things I recommend. Did you use `prettier`, or did you do this manually? I didn't see prettier in your `package.json` (which is where the project dependencies are listed)

Reply:
I am pretty sure I used Prettier. I will have to check and see why it wasn’t in package.json